moon/skyline.py

```diff
@@ -0,0 +1,100 @@
+"""
+  Moon Skyline analysis
+"""
+import math
+
+import numpy as np
+import cv2  # for debug
+
+
+def skyline(img):
+    b = img[:,:,0]
+    g = img[:,:,1]
+    r = img[:,:,2]
+    mask = r > 10
+    return mask.argmax(axis=0)
+
+
+def find_peaks(arr):
```

Comment:
It is not obvious that this function looks for at most two peaks. It needs either a better name, or a comment.

Reply:
this PR is draft, i.e. still work in progress, looking for suggestions ... and at the end I want to find as many reliable peaks as possible